pkg/controller/template_controller.go

```diff
@@ -0,0 +1,31 @@
+package controller
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	"github.com/operator-framework/combo/api/v1alpha1"
+)
+
+// +kubebuilder:rbac:groups=core.rukpak.io,resources=provisionerclasses,verbs=get;list;watch;create;update;patch;delete
```

Comment:
This is a copy-paste left over from the rukpak repository. This isn't super important currently, but we'll want to update this to use the combo group and the template custom resource (vs. the provisionerclass resource currently specified).

Reply:
This may actually not be needed at all as these permissions are all provided via the `Role` and `Rolebinding` for in the `deployment.yaml`. What do you think though?

Reply:
Those are used to help inform resource generation: https://github.com/kubernetes-sigs/kubebuilder/blob/master/docs/kubebuilder_annotation.md. If we're supplying our own RBAC definitions vs. using what kubebuilder scaffolded out for us, I think we can drop the RBAC annotations entirely from the controller(s) packages.

Reply:
Got it, in that case I'll remove this comment.